data/data/gcp/main.tf

```diff
@@ -47,8 +47,9 @@ module "master" {
   subnet         = module.network.master_subnet
   zones          = distinct(var.gcp_master_availability_zones)
 
-  root_volume_size = var.gcp_master_root_volume_size
-  root_volume_type = var.gcp_master_root_volume_type
+  root_volume_size         = var.gcp_master_root_volume_size
+  root_volume_type         = var.gcp_master_root_volume_type
+  root_volume_kms_key_link = var.gcp_root_volume_kms_key_link
```

Comment:
Need to add this to the "bootstrap" module, too.

https://github.com/openshift/installer/pull/4397/files#diff-c9542c57acc94c4d250fd8d806ac9cf57c9ccd52c3c8e282a34527198949ff57R35

Reply:
Is that not done as the change in `data/data/gcp/bootstrap/main.tf`, or does that need to go into `data/data/bootstrap` somewhere?

Reply:
In `/data/data/gcp/main.tf` as linked above. It is currently only passing the variable from main to master, but needs to do the same for bootstrap.